jasmine-standalone-3.6.0 (5)/spec/gameSpec.js

```diff
@@ -90,4 +90,16 @@ describe("Frame ", function() {
     })
   })
 
+  describe('spare', function(){
+    it('knows when it is a spare', function(){
+      frame.enterFirstRollScore(10);
+      expect(frame.spare()).toEqual(true)
```

Comment:
Isn't a score of 10 a strike, not a spare? A spare is when the two scores form the frame add up to 10 I thought